sdk/src/main/java/com/mapbox/maps/MapboxMapOptions.kt

```diff
@@ -58,6 +59,22 @@ data class MapboxMapOptions constructor(
    */
   var textureView: Boolean
 
+  /**
+   * Default plugins to initialize the map with
+   */
+  var plugins = arrayOf(
```

Comment:
So if user specifies nothing - all the plugins will be added.
If user mutates `plugins` - only those will be applied that are specified.
Several questions:
1. Do we still want to have some plugins as mandatory?
2. Could it be a good idea to provide also list of __excluded__ plugins so that it will be easy to enable everything except e.g. one plugin. In that case final resulting plugin list will be `enabled_plugins_list - disabled_plugins_list`

Reply:
> Do we still want to have some plugins as mandatory?

Not at this time, long term we will be using a feature flag system from common sdk.
That system will allow us to specify if a user needs to have a certain plugin enabled or is authorized to disable it. 

> Could it be a good idea to provide also list of excluded plugins so that it will be easy to enable everything except e.g. one plugin. In that case final resulting plugin list will be enabled_plugins_list - disabled_plugins_list

Makes sense to me for supporting that!